server/routes/workspaceRoutes.js

```diff
@@ -2,13 +2,15 @@ import express from "express";
 import {
   createWorkspace,
   deleteWorkspace,
+  editWorkspace,
   shareWorkspace,
 } from "../controllers/workspaceController.js";
 import { protect } from "../middleware/authMiddleware.js";
 
 const router = express.Router();
 
 router.route("/").post(protect, createWorkspace);
+router.route("/:id").put(protect, editWorkspace);
 router.route("/:id").delete(protect, deleteWorkspace);
```

Comment:
Let's change it to this
```suggestion
router.route("/:id").put(protect, editWorkspace).delete(protect, deleteWorkspace);
```